spec/requests/admin/dashboard_controller_spec.rb

```diff
@@ -172,12 +172,15 @@ def populate_new_features(date1 = nil, date2 = nil)
   end
 
   describe '#new_features' do
+    after do
+      Discourse.redis.del "new_features_last_seen_user_#{admin.id}"
+      Discourse.redis.del "new_features"
+      Discourse.redis.del "last_viewed_feature_dates_for_users_hash"
```

Comment:
maybe we should have a reset function on `DiscourseUpdates` something that wouldn't even need the admin id and would just erase all matching keys.

Reply:
I agree with this comment. I believe this is what we usually do.

Reply:
done!